docs/decorators/withTheme.tsx

```diff
@@ -47,16 +52,16 @@ function SetBackground({ viewMode, id }: { viewMode: string; id: string }) {
         }
       `
     );
-  }, [selectorId, selector, theme]);
+  }, [selectorId, selector, mode, theme]);
 
   return null;
 }
 
 export const withTheme: DecoratorFn = (StoryFn, context) => {
-  const { density, theme } = context.globals;
+  const { density, mode } = context.globals;
 
-  if (theme === "side-by-side" || theme === "stacked") {
-    const isStacked = theme === "stacked";
+  if (mode === "side-by-side" || mode === "stacked") {
```

Comment:
I thought mode can only be 'light' or 'dark'. Is this change intended?

Reply:
This is a storybook only thing. I don't really think the name here actually matters